lte/protos/policydb.proto

```diff
@@ -131,6 +130,12 @@ message FlowMatch {
   }
   Direction direction = 8;
   string app_name = 9;
+  IPAddress ip_src = 10;
+  IPAddress ip_dst = 11;
```

Comment:
would you mind adding a comment of what is src and dst?  

Or maybe, if it makes sense, changing it  to something like `ip_uplink` `ip_downlink`

Reply:
Flow match doesn't have a notion of uplink/downlink. It's just matching on ip src or ip dst